docs/deployment/aks.md

```diff
@@ -100,7 +100,7 @@ IP address to the `kong-proxy` Service so please be patient.
       kubernetes.io/ingress.class: "nginx"
   spec:
     rules:
-      - host:
+      - host: dummy.kong.example
```

Comment:
@alx8 Any reason this change is necessary?
Kong can route without a Host header and the Ingress spec allows for an empty Host as well.